gtk2/0001-Work-around-crasher-when-closing-a-window.patch

```diff
@@ -0,0 +1,35 @@
+From 07e48efbe8fe599d736b0142ca211ee5789ebef1 Mon Sep 17 00:00:00 2001
+From: Bastien Nocera <hadess@hadess.net>
+Date: Mon, 31 Oct 2022 12:42:43 +0100
+Subject: [PATCH] Work-around crasher when closing a window
+
+See glib patch
+---
+ gtk/gtkwidget.c | 4 ++++
+ 1 file changed, 4 insertions(+)
+
+diff --git a/gtk/gtkwidget.c b/gtk/gtkwidget.c
+index 1d1f6bb60e..ed70caab8e 100644
+--- a/gtk/gtkwidget.c
++++ b/gtk/gtkwidget.c
+@@ -3155,6 +3155,8 @@ gtk_widget_unparent (GtkWidget *widget)
+   g_object_unref (widget);
+ }
+ 
++void g_set_crash_workaround(gpointer p);
++
+ /**
+  * gtk_widget_destroy:
+  * @widget: a #GtkWidget
+@@ -3179,6 +3181,8 @@ gtk_widget_destroy (GtkWidget *widget)
+ {
+   g_return_if_fail (GTK_IS_WIDGET (widget));
+ 
++  if (g_strcmp0 (G_OBJECT_TYPE_NAME (widget), "GtkMenuBar") == 0)
```

Comment:
```suggestion
+  if (GTK_IS_MENU_BAR (widget))
```

Reply:
That would mean including `gtkmenubar.h` into `gtkwidget.c`, which is what I was trying to avoid. It's probably faster though, so I'll do that.